docker/docker-compose.yml

```diff
@@ -32,10 +44,31 @@ services:
     volumes:
       - '../frontend:/app'
       - '/app/node_modules'
-    ports:
-      - '3000:3000'
     links:
       - database
+    ports:
+      - "82:82"
     environment:
+      - HOST=${HOST}
+      - PORT=82
       - NODE_ENV=development
       - CHOKIDAR_USEPOLLING=true
+
+
+
+  proxy:
```

Comment:
very inconsistent formatting in this file. Too many unnecessary new lines throughout the whole file.